lib/v2/screens/profile_screens/profile/components/citizenship_card.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/datasource/remote/model/profile_model.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/i18n/profile.i18n.dart';
+import 'package:seeds/v2/navigation/navigation_service.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/v2/screens/profile_screens/profile/interactor/viewmodels/bloc.dart';
+
+class CitizenshipCard extends StatelessWidget {
+  final ProfileModel profile;
+
+  const CitizenshipCard({
+    Key? key,
+    required this.profile,
+  }) : super(key: key);
```

Comment:
1 line instead 4 and also I suggest not use named parameter for just one parameter.

`const CitizenshipCard(this.profile, {Key? key}): super(key: key);`

Reply:
fix